pkg/store/ocistore/session_store.go

```diff
@@ -194,3 +194,30 @@ func (s *OCIStore) UpdateSessionExpiresAt(id string, expiresAt time.Time) error
 	}
 	return nil
 }
+
+func (s *OCIStore) DeleteExpiredSessions() error {
+	secret, err := s.getSessionSecret()
+	if err != nil {
+		return errors.Wrap(err, "failed to get session secret")
+	}
+
+	updateSessionSecret := false
+	for id, data := range secret.Data {
+		session := sessiontypes.Session{}
+		if err := json.Unmarshal(data, &session); err != nil {
+			return errors.Wrap(err, "failed to unmarshal session")
```

Comment:
could we `continue` here instead, so as to still purge the parsable sessions?

with a log, of course